js/apps/admin-ui/src/identity-providers/add/DetailSettings.tsx

```diff
@@ -101,6 +109,44 @@ const Header = ({ onChange, value, save, toggleDeleteDialog }: HeaderProps) => {
     },
   });
 
+  const importSamlKeys = async (
+    providerId: string,
+    entityDescriptorUrl: string,
+  ) => {
+    try {
+      const result = await adminClient.identityProviders.importFromUrl({
+        providerId: providerId,
+        fromUrl: entityDescriptorUrl,
+      });
+      if (result["signingCertificate"]) {
+        setValue(`config.signingCertificate`, result["signingCertificate"]);
+        addAlert(t("importKeysSuccess"), AlertVariant.success);
+      } else {
+        addError(
+          "importKeysError",
+          "signingCertificate is not defined in the metadata",
```

Comment:
Ideally the error message is also localized.

Reply:
Done!